### PR TITLE
Fix/postgres password

### PR DESCRIPTION
**Pull request by fmubaidien**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by fmubaidien** [comment]
This resolves issue #46

**Comment by banzo** [comment]
@fmubaidien thank you, could you update the README.md accordingly?

File: values.yaml [hunk 1]
```diff
@@ -177,7 +177,7 @@ zabbixweb:
   # -- User of database
   POSTGRES_USER: zabbix
   # -- Password of database
-  POSTGRES_PASSWORD: zabbix_pwd
+  POSTGRES_PASSWORD: zabbix
```
Comment (aeciopires) [inline values.yaml:180]: Hi @fmubaidien!

Thanks for your contributition.

The problem is the line 59 of values.yaml file.

The password must the same in parameters:

postgresql.auth.password 
zabbixserver.POSTGRES_PASSWORD
zabbixweb.POSTGRES_PASSWORD

In the case, you need change only the ``postgresql.auth.password``  parameter to ``zabbix_pwd``

**Review by aeciopires — CHANGES_REQUESTED** [review]

**Comment by fmubaidien** [comment]
I forgot to change server but I changed it now in order to keepy postgres user and the zabbix user as two different passwords, readme has also been changed

**Comment by fmubaidien** [comment]
Feel free to close the PR if you would rather do it in a different way, either way thanks for the helm chart

**Comment by aeciopires** [comment]
Great @fmubaidien!

This PR is cool for me.

Thanks again.

**Pull request merged**